### DISCUSSION TITLE
Is it possible to prepopulate DB on 1st start?

### DISCUSSION THREAD

**Discussion by laszlo1337** [pocketbase/pocketbase#3875] [Q&A]
Hello. My app relies on some types, statuses etc which have their own tables. Currently I'm developing locally but when I deploy I'd want those to be there already. 
Is it possible to populate these tables when db is first started?

I'm using standalone PB v.0.19.3 with JS pb 0.19.0

**Comment by ganigeorgiev** [comment]
Yes, use migrations:

- JS: https://pocketbase.io/docs/js-migrations/
- Go: https://pocketbase.io/docs/go-migrations/

If you use the prebuilt executable, `--automigrate` is enabled by default and there should be a `pb_migrations` dir containing your db schema changes.
New/unapplied migrations are automatically applied on `serve`, aka. it is enough the other environment to have the `pb_migrations` to share the same schema as your local one.
